range.c

```diff
@@ -778,14 +778,16 @@ sym_each_i(VALUE v, VALUE arg)
 
 /*
  *  call-seq:
- *     rng.size                   -> num
+ *    size -> non_negative_integer
```

Comment:
`Infinity` (used in the example below) is not a non-negative integer, nor is `nil`.  Maybe `integer | nil | Infinity`? Other approaches may work.

Reply:
Fixed ('or', not '|', as elsewhere.